<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="MyDemoWidget" name="MyWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>621</width>
      <height>131</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777213</height>
     </size>
    </property>
   </widget>
   <widget class="QTabWidget" name="mainTab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>171</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Priority</string>
     </attribute>
     <widget class="QPushButton" name="samePriorityButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Priority</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>T1</string>
      </property>
      <widget class="QRadioButton" name="highPrioT1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>High</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="medPrioT1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Med</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="lowPrioT1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Low</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>T2</string>
      </property>
      <widget class="QRadioButton" name="lowPrioT2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Low</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="medPrioT2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Med</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="highPrioT2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>High</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>T3</string>
      </property>
      <widget class="QRadioButton" name="highPrioT3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>High</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="lowPrioT3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Low</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="medPrioT3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Med</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="startThreadsWithCriticalSectionButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Critical section</string>
      </property>
     </widget>
     <widget class="QPushButton" name="startThreadsWithMutexButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string> Unnamed mutex</string>
      </property>
     </widget>
     <widget class="QPushButton" name="startProcessWithNamedMutexButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Named mutex</string>
      </property>
     </widget>
     <widget class="QPushButton" name="startThreadsWithSemaphoreButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>110</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Semaphore</string>
      </property>
     </widget>
     <widget class="QPushButton" name="startProcessWithNamedSemaphoreButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>70</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Named semaphore</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Events</string>
     </attribute>
     <widget class="QPushButton" name="startManualEventsButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>271</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start with manual event</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopManualEventsButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="continueManualEventsButton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Continue</string>
      </property>
     </widget>
     <widget class="QPushButton" name="startProcessWithNamedEventButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>271</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start with named event</string>
      </property>
     </widget>
     <widget class="QPushButton" name="startAutoEventsButton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>251</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start with auto event</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopManualEventsButton2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="continueManualEventsButton2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>40</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Continue</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>mainTab</zorder>
   <zorder>MyWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyDemoWidget</class>
   <extends>QWidget</extends>
   <header location="global">mydemowidget.h</header>
   <container>1</container>
   <slots>
    <slot>slotAddPoint(MyPoint2)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
